src/main/scala/org/broadinstitute/hail/variant/Variant.scala

```diff
@@ -266,9 +266,11 @@ case class Variant(contig: String,
   def locus: Locus = Locus(contig, start)
 
   def isAutosomalOrPseudoAutosomal: Boolean =
-    !isMitochondrial && !isX && !isY || inXPar || inYPar
+    (!isMitochondrial && !isX && !isY) || inXPar || inYPar
 
-  def isMitochondrial = contig == "m" || contig == "M" || contig == "MT" || contig == "26"
+  def isAutosomal = !isMitochondrial && !isX && !isY
+
+  def isMitochondrial = contig.toUpperCase == "M" || contig.toUpperCase == "MT" || contig == "26"
```

Comment:
I think this will do contig.toUpperCase twice.  How about:

`def isMitochondrial = val c = contig.toUpperCase; c == "M" || c == "MT" || contig == "26"`

If "MT" is more common, put that case first.